qlib/config.rs

```diff
@@ -138,4 +140,26 @@ pub enum CudaMemType {
     Default,
     UM, //Unified Memory
     MemPool,
+}
+
+#[derive(Clone, Copy, Debug, PartialOrd, Ord, Eq, PartialEq, Serialize, Deserialize)]
+pub enum CCMode {
+    None,
+    Normal,
+    NormalEmu,
+    SevSnp,
+    Max
+}
+
+impl CCMode {
+    pub fn from(value: u64) -> Self {
```

Comment:
why mapping from numeric values? Aren't this enum user defined?

Reply:
Used for reusing the sharedspace u64 passed to the kernel as the CCMode when cc is enabled.